config/ackermann_config.lua

```diff
@@ -8,3 +12,8 @@ ak_angular_error_bias = DegToRad(0);
 ak_angular_error_rate = 0.1;
 
 ak_drive_callback_topic = "/ackermann_curvature_drive"
+
+-- Car configuration
+arkermann_num_segments = 20
```

Comment:
ackermann drive, not arkermann. This might as well be correct everywhere it is used.

Reply:
done